plugins/wordpress-plugin/gatsby-node.js

```diff
@@ -248,7 +251,7 @@ exports.createSchemaCustomization = async ({ actions }) => {
 
   // WordPress types
   actions.createTypes(/* GraphQL */ `
-    type WpMediaItem implements Node & HomepageImage {
+    type WpMediaItem implements Node & RemoteFile & HomepageImage {
```

Comment:
While the starters do not use this (due to lack of support in the other CMSs), this should ensure image CDN can be used for any user-written queries